microlith/loki/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -273,3 +273,21 @@ groups:
           title: "Large Number of Dropped Ticks Detected (node: {{ $labels.hostname }})"
           description: Couchbase Server nodes regularly send heartbeat ticks to each other. Dropped ticks are usually a sign of resource contention.
           remediation: Increase number of CPUs available to Couchbase Server or, if you are running a virtualised environment, check for VM overcommitment.
+
+      - alert: CB90067-serviceRollback
+        expr: |
+          count_over_time( {file="memcached.log"} |= "requires rollback to seqno:0" != "eq_dcpq:replication" [1h]) > 0
```

Comment:
May be worth mentioning that the warning is if this has occurred in the past hour somewhere in the checker documentation?

Reply:
I can do that :)